packages/ui/primitives/react-hooks/src/useDefaultValue.ts

```diff
@@ -0,0 +1,28 @@
+//
```

Comment:
could you take a look at common/react-async which has a bunch of similar hooks (e.g., useStateRef). "async" isn't a good place for these since they are general purpose, so perhaps they need to be transplanted here. Would be good for you to review these.

Reply:
Great idea.